components/table/table-col-sort-button.js

```diff
@@ -122,8 +141,18 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 		const iconView = !this.nosort ?
 			html`<d2l-icon icon="${this.desc ? 'tier1:arrow-toggle-down' : 'tier1:arrow-toggle-up'}"></d2l-icon>` :
 			null;
+		const buttonTitle = this._getSortButtonTitle();
+		const description = this.localize(!this.nosort ? 'components.table.change-sort-order' : 'components.table.add-sort-order');
+
+		return html`<button aria-description="${description}" aria-label="${ifDefined(buttonTitle)}" title="${buttonTitle}" type="button"><slot></slot>${iconView}</button>`;
```

Comment:
In the spec is there both `aria-label` and `title`? I had thought we just wanted `title`